website/templates/project/statistics.mako

```diff
@@ -5,6 +5,23 @@
   <h2 class="text-300">Analytics</h2>
 </div>
 
+<div id="hiddenFlag"></div>
+<script src="/static/js/ads.js"></script>
+<script>
+  if( window.canRunAds === undefined ){
+    var banner = document.createElement('div');
+    banner.className += 'm-b-md p-md osf-box-lt box-round';
+    var node = document.createTextNode("The use of adblocking software may prevent site analytics from loading properly. For more information go");
+    banner.appendChild(node);
+    var link = document.createElement('a');
+    link.href = "https://openscience.atlassian.net/browse/NCP-821";
```

Comment:
This link won't be useful for anyone who doesn't work here (try opening it in an incognito window to see). Check through http://help.osf.io to see if there's a page you can link to there, and if not, ask @saradbowman if she has a recommended place to link.